client/src/notification/NotificationToaster.js

```diff
@@ -5,21 +5,23 @@
 
 import React, { Fragment, useEffect } from 'react';
 import toast from 'react-hot-toast';
-import { useSelector } from 'react-redux';
+import { useSelector, useDispatch } from 'react-redux';
+import { CLEAR_NOTIFICATION } from '../actions/types';
 export const NotificationToaster = () => {
   const { error, success } = useSelector((state) => state.notification);
+  const dispatch = useDispatch();
 
   useEffect(() => {
-    if(error){
+    if (error) {
       toast.error(error.data);
     }
-  }, [error])
-
-  useEffect(() => {
-    if(success){
+    if (success) {
       toast.success(success);
     }
-  }, [success])
-  
+    dispatch({
+      type: CLEAR_NOTIFICATION,
+    });
+  }, [error, success]); // eslint-disable-line react-hooks/exhaustive-deps
+
```

Comment:
This was added by @pmyip3 after we found out that the toast didn't show up if the message was the same due to useEffect()